src/psij/web/js/localStorage.js

```diff
@@ -0,0 +1,79 @@
+//  Global Storage, shortcut for sqs.
```

Comment:
I needed persistence to prevent IntroBlurbs from showing up after the user has closed them.  We can use it for many purposes.

Reply:
That sounds unusual in that I'm not familiar with any sites that do this except in very special cases when required by law or some such (like cookies preferences). But those are quite annoying and intrusive. We could simply have a brief introductory text that users can ignore after reading it once and, in case they forget it, read it again.

Additionally, the code already has a library that saves settings in cookies, which is used to remember the summary view type and other settings.